tests/libcxx/expected_results.txt

```diff
@@ -89,6 +89,8 @@ std/utilities/format/format.functions/vformat.pass.cpp FAIL
 # libc++ doesn't implement P2321R2's changes to vector<bool>::reference
 std/containers/sequences/vector.bool/iterator_concept_conformance.compile.pass.cpp FAIL
 
+# libc++ hasn't updated move_iterator for P2520R0
+std/iterators/predef.iterators/move.iterators/move.iterator/types.pass.cpp FAIL
 

```

Comment:
Nitpick, not worth resetting testing by itself: `skipped_tests.txt` is preserving the 2 newlines that were here, but `expected_results.txt` isn't. This doesn't *seriously* complicate diffing, but we should try to keep these files in sync as much as possible.